dotnet-desktop-guide/framework/wpf/app-development/wpf-xaml-browser-applications-overview.md

```diff
@@ -16,7 +16,7 @@ ms.assetid: 3a7a86a8-75d5-4898-96b9-73da151e5e16
 # WPF XAML Browser Applications Overview
 
 <a name="introduction"></a>
-XAML browser applications (XBAPs) combines features of both Web applications and rich-client applications. Like Web applications, XBAPs can be deployed to a Web server and started from Internet Explorer or Firefox. Like rich-client applications, XBAPs can take advantage of the capabilities of WPF. Developing XBAPs is also similar to rich-client development. This topic provides a simple, high-level introduction to XBAP development and describes where XBAP development differs from standard rich-client development.
+XAML browser applications (XBAPs) combines features of both Web applications and rich-client applications. Like Web applications, XBAPs can be deployed to a Web server and started from Internet Explorer Desktop Application or Firefox Desktop Application. Like rich-client applications, XBAPs can take advantage of the capabilities of WPF. Developing XBAPs is also similar to rich-client development. This topic provides a simple, high-level introduction to XBAP development and describes where XBAP development differs from standard rich-client development.
```

Comment:
```suggestion
XAML browser applications (XBAPs) combines features of both Web applications and rich-client applications. Like Web applications, XBAPs can be deployed to a Web server and started from Internet Explorer or Firefox on Windows. Like rich-client applications, XBAPs can take advantage of the capabilities of WPF. Developing XBAPs is also similar to rich-client development. This topic provides a simple, high-level introduction to XBAP development and describes where XBAP development differs from standard rich-client development.

>[!IMPORTANT]
> XBAPs require legacy browsers to operate, such as Internet Explorer and Firefox. These older browsers are usually unsupported on Windows 10 and Windows 11.
```

Reply:
That is even better!  I really like the "important" note, it spells out the reality rather concisely.